hw_1/Collection - dna converter.py

```diff
@@ -0,0 +1,78 @@
+# dna/rna convert program
+
+# dictionary for complement, transcription reverse transcription
+dict_complement = {'A': 'T', 'T': 'A', 'C': 'G', 'G': 'C',
+                   'a': 't', 't': 'a', 'c': 'g', 'g': 'c'}
+dict_transcript = {'A': 'U', 'T': 'A', 'C': 'G', 'G': 'C',
+                   'a': 'u', 't': 'a', 'c': 'g', 'g': 'c'}
+dict_reverse_trans = {'A': 'T', 'U': 'A', 'C': 'G', 'G': 'C',
+                      'a': 't', 'u': 'a', 'c': 'g', 'g': 'c'}
+
+
+# function for complement, transcription and reverse transcription
+def foo_common(seq, dictionary):
+    output = str()
+    for j in range(len(seq)):
+        output += dictionary[seq[j]]
```

Comment:
В питоне можно итерироваться прямо по строке
```suggestion
    for nucl in seq:
        output += dictionary[nucl]
```